js/components/user/delete-user-modal.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+<modal :title="_('Confirm deletion')"
+    class="modal-danger"
+    v-ref:modal>
+
+    <div class="modal-body">
+        <p class="lead text-center">
+            {{ _('You are about to delete the profile of the user {first_name} {last_name}.',
```

Comment:
Might be shorter and easier to maintain with `fullname` instead of `first_name` and `last_name`.